pom.xml

```diff
@@ -129,11 +134,9 @@
     </profile>
     <profile>
       <id>spark-3.2</id>
-      <activation>
```

Comment:
When profile java-8 is activated, all profiles that are active by default are automatically deactivated.
As a result, `delta.package.name` is undefined and we get a invalid artifactId in delta dependency.

Ref: https://maven.apache.org/guides/introduction/introduction-to-profiles.html#details-on-profile-activation